### PR TITLE
chore: CS fixes

### PR DESCRIPTION
**Pull request by keradus**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix CS
| License       | MIT

Few steps closer to clean CS (`php-cs-fixer fix` not reporting any issues)

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/CoverageListener.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class CoverageListener implements TestListener
     public function __construct(?callable $sutFqcnResolver = null, bool $warningOnSutNotFound = false)
     {
         $this->sutFqcnResolver = $sutFqcnResolver ?? static function (Test $test): ?string {
-            $class = \get_class($test);
+            $class = $test::class;
```
Comment (derrabus) [inline src/Symfony/Bridge/PhpUnit/CoverageListener.php]: We can't apply this fixer to the PHPUnit bridge. We still need to support PHPUnit 7 here.

Comment (keradus) [reply src/Symfony/Bridge/PhpUnit/CoverageListener.php]: ack, any plans to drop support for legacy versions ?

**Comment by keradus** [comment]
- 8.4 failing on core repo as well
- psalm complains about `src/Symfony/Bridge/PhpUnit`, i saw it in previous PRs of mine too and they were not related to my code before, I'm happy to ignore them or have anyone explaining me how i broke it, i tried to debug psalm few times without success

File: src/Symfony/Component/Serializer/Tests/Normalizer/PropertyNormalizerTest.php [hunk 2]
```diff
@@ -451,19 +451,19 @@ public function testMultiDimensionObject()
     {
         $normalizer = $this->getDenormalizerForTypeEnforcement();
         $root = $normalizer->denormalize([
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Serializer/Tests/Normalizer/PropertyNormalizerTest.php]: I'd rather put the opening bracket on the next line

File: src/Symfony/Component/Form/Tests/Util/StringUtilTest.php [hunk 3]
```diff
@@ -56,7 +56,7 @@ public static function spaceProvider(): array
             ['0020'],
             ['00A0'],
             ['1680'],
-//            ['180E'],
+            //            ['180E'],
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Tests/Util/StringUtilTest.php]:
```suggestion
            //['180E'],
```

File: src/Symfony/Bridge/PhpUnit/CoverageListener.php (hunk 1 above)
Comment (derrabus) [reply src/Symfony/Bridge/PhpUnit/CoverageListener.php]: Plans, yes. 🙂

File: src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php [hunk 4]
```diff
@@ -146,12 +146,12 @@ public function testExtraction(iterable|string $resource)
             ],
             'validators' => [
                 'message-in-constraint-attribute' => 'prefixmessage-in-constraint-attribute',
-//                'custom Isbn message from attribute' => 'prefixcustom Isbn message from attribute',
+                //                'custom Isbn message from attribute' => 'prefixcustom Isbn message from attribute',
```
Comment (gharlan) [inline src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php]:
```suggestion
                // 'custom Isbn message from attribute' => 'prefixcustom Isbn message from attribute',
```

File: src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php [hunk 5]
```diff
@@ -146,12 +146,12 @@ public function testExtraction(iterable|string $resource)
...
                 'custom Isbn message from attribute with options as array' => 'prefixcustom Isbn message from attribute with options as array',
                 'custom Length exact message from attribute from named argument' => 'prefixcustom Length exact message from attribute from named argument',
                 'custom Length exact message from attribute from named argument 1/2' => 'prefixcustom Length exact message from attribute from named argument 1/2',
                 'custom Length min message from attribute from named argument 2/2' => 'prefixcustom Length min message from attribute from named argument 2/2',
-//                'custom Isbn message' => 'prefixcustom Isbn message',
+                //                'custom Isbn message' => 'prefixcustom Isbn message',
```
Comment (gharlan) [inline src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php]:
```suggestion
                // 'custom Isbn message' => 'prefixcustom Isbn message',
```

File: src/Symfony/Component/Form/Tests/Util/StringUtilTest.php (hunk 3 above)
Comment (keradus) [reply src/Symfony/Component/Form/Tests/Util/StringUtilTest.php]: manually adjusted

File: src/Symfony/Component/Serializer/Tests/Normalizer/PropertyNormalizerTest.php (hunk 2 above)
Comment (keradus) [reply src/Symfony/Component/Serializer/Tests/Normalizer/PropertyNormalizerTest.php]: manually adjusted

File: src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php (hunk 4 above)
Comment (keradus) [reply src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php]: manually adjusted

File: src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php (hunk 5 above)
Comment (keradus) [reply src/Symfony/Component/Translation/Tests/Extractor/PhpAstExtractorTest.php]: manually adjusted

**Review by nicolas-grekas — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by fabpot** [comment]
For 7.2.

**Comment by nicolas-grekas** [comment]
Thank you @keradus.

**Pull request merged**